perf/index.js

```diff
@@ -13,5 +14,6 @@ test('Large item', function (t) {
     var end = new Date()
     totalDuration += end - start
   }
-  t.ok(totalDuration < 10, 'wanted ' + totalDuration + 'ms to be less than 10ms')
+  var maxTime = iterations * 0.05 // 0.05ms per result
```

Comment:
A precision of 1 would be exactly the same. It went up from 6ms per 500 to 10ms per 500 on travis. It's still 6ms locally, but travis complained.